mdpow/config.py

```diff
@@ -167,8 +169,13 @@ def get(self, section, option):
            Prior versions would convert case-insensitively (e.g. "NONE"
            and "none")
         """
-        value = self.conf[section][option]
-        return value if value != "None" else None
+
+        value = self.conf[section]
+        if isinstance(value, dict):
```

Comment:
Under which conditions is it necessary to check if value is a dict? Did the code fail?

If this is a "drive-by" code improvement, put it in a separate PR. 

If it must be here then rather "ask for forgiveness than permission" with the more Pythonic
```python
try:
   value = self.conf[section][option]
except TypeError:
   value = None
return value
```
Has higher performance (although that doesn't matter here), is more robust (can be any mapping), and most Python coders would also call it more readable.